lib/card.js

```diff
@@ -0,0 +1,112 @@
+export function createCardComponent(cardData) {
+  const section = createSection();
+
+  const headingElement = createHeading(cardData);
+
+  const questionElement = createQuestion(cardData);
+
+  const buttonElement = createButton(cardData);
+
+  const answerElement = createAnswer(cardData);
```

Comment:
it might be the case, that you have to create the answerElement first, because you need to declare it before the button function tries to use it.

Reply:
only change line 8 & 10.